src/main/resources/messages.properties

```
@@ -20,6 +20,11 @@ site.action=Action
 site.select=Please select
 site.useThisRecord=Use this record
 site.print=Print this page
+site.manyResults={0} - too many results
```

I think this needs the extra description
```suggestion
site.tooManyResults={0} - too many results
```